--- origsrc/SGMLSpm/Makefile.PL	2009-10-27 01:17:58.692205900 -0500
+++ src/SGMLSpm/Makefile.PL	2009-10-27 01:24:56.766385700 -0500
@@ -10,8 +10,8 @@ WriteMakefile(
                     'skel.pl' => '$(INST_LIBDIR)/sgmlspl-specs/skel.pl',
                 },
       'MAN3PODS'=> { 'SGMLS.pm' => '$(INST_MAN3DIR)/SGMLS.3pm',
-                    'Output.pm' => '$(INST_MAN3DIR)/SGMLS::Output.3pm',
-                    'Refs.pm' => '$(INST_MAN3DIR)/SGMLS::Refs.3pm',
+                    'Output.pm' => '$(INST_MAN3DIR)/SGMLS.Output.3pm',
+                    'Refs.pm' => '$(INST_MAN3DIR)/SGMLS.Refs.3pm',
                 },
       'EXE_FILES' => [ 'sgmlspl', ],
 );
